packages/flutter_tools/lib/src/build_info.dart

```diff
@@ -11,14 +11,15 @@ import 'globals.dart';
 /// Information about a build to be performed or used.
 class BuildInfo {
   const BuildInfo(this.mode, this.flavor,
-      {this.previewDart2,
+      {this.previewDart2: false,
```

Comment:
while we're here let's change this to flutter style, which moves the { to the previous line and indents by 2 rather than 4.

(I think dartfmt will do this by default if you put a comma at the end of the last argument.)

Reply:
Done.